### ISSUE TITLE
Request

### ISSUE THREAD

**Issue by snomann72** [conan513/SingleCore_TC#62]
As you may recall, someone made a SQL File that is called Trinity Mall Vendors.  Since the TrinityCore been updated so many times, Trinity Mall Vendors doesn't work with 3.3.5a anymore.  Wanted to ask is it possible to add it to TC-SPP or would it break the repack?

**Comment by conan513** [comment]
I think it's possible but need to check it

**Comment by snomann72** [comment]
I found a updated version to Trinity Mall Vendors.  The install.bat file is broken tho.  It can be manually added to the world files.  Don't have any issues when I just tested it.  The only problem is, if a person wants items, there is no portal there, and it would need to be added to the portal npc.  Since the AHBot don't sell everything, the Trinity Mall Vendors has the solution.  The latest version, which is 3.7 Update works the best.  The ones that is lower don't work with the latest TC update.

It has portals leaving the GMIsland, which is good.  Might add a portal NPC there so they can port to a area that they can go and quest.
